include/mo.php

```diff
@@ -74,4 +74,24 @@ public function import_from_db( $lang ) {
 	public function delete_entry( $string ) {
 		unset( $this->entries[ $string ] );
 	}
+
+	/**
+	 * Translates a string or returns false if the translation is not found.
+	 * Contrary to `self::translate()`, this method doesn't fallback to the source string.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string $source The source string to translate.
+	 * @return string|false The translated string or false if not found.
+	 */
+	public function translate_or_fail( string $source ) {
```

Comment:
I'm not fond of the proposed name. `maybe_translate()`? `translate_if_exists()`? `translate_if_any()`?

Reply:
Done in 813cda54.